components/Home.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+const api = axios.create({
+  baseURL: "https://chriss-new-app.onrender.com",
+});
+
+
+
+export const Home = () => {
+    const [articles, setArticles] = useState([])
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() =>{
+        api.get('/api/articles')
+        .then((response) => {
+            console.log(response.data.articles)
+            setArticles(response.data.articles)
+            setLoading(false);
+        })
+    }, [])
+    if(loading) {
+        return <div>Loading!</div>
+    }
+
+    return <div className="home">
+        <h2>Articles</h2>
+        <h3>Sort By Topics/Authors</h3>
+        
+        <ul>
+            {articles.map((article) => {
+                return <li className="article-list-item">
+                    <div className="article-li-title-info">
+                        <h4>{article.title}</h4>
+                        <div className="aritcle-list-item-info">
+                            <button>Topic: {article.topic}</button>
```

Comment:
I'm a little unsure as to why these are buttons here. Buttons have a reason to exist, they should implement some functionality - ie they shouldn't be used for styling purposes.